docker-compose/Makefile

```diff
@@ -17,26 +17,37 @@ DOCKER_IMAGE_PREFIX ?= openwhisk
 PROJECT_HOME ?= ./openwhisk-master
 WSK_CLI ?= $(PROJECT_HOME)/bin/wsk
 
+DOCKER_NETWORK ?= "openwhisk_default"
+
 OPEN_WHISK_DB_PREFIX ?= local_
 
 DOCKER_KERNEL ?= $(shell docker version --format "{{.Server.KernelVersion}}")
 ifeq ("$(UNAME_STR)","Linux")
 	RUNC_BINARY   ?= $(shell (which runc || which docker-runc))
 	DOCKER_BINARY ?= $(shell (which docker))
-else 
+else
 	RUNC_BINARY   ?= $(shell if [[ $(DOCKER_KERNEL) == *-moby || $(DOCKER_KERNEL) ==  *-boot2docker ]]; then (docker run --rm --privileged --pid=host debian nsenter -t 1 -m -u -n -i sh -c "which runc || which docker-runc"); else (which runc || which docker-runc); fi)
 	DOCKER_BINARY ?= $(shell if [[ $(DOCKER_KERNEL) == *-moby || $(DOCKER_KERNEL) ==  *-boot2docker ]]; then (docker run --rm --privileged --pid=host debian nsenter -t 1 -m -u -n -i sh -c "which docker"); else (which docker); fi)
 endif
 ifndef VERBOSE
 .SILENT:
 endif
 
+.PHONY: deploy
```

Comment:
is the intent to run this after `quick-start` ?

Reply:
It's meant to be run instead of `quick-start` to set up a fully-fledged environment without `hello-world` and with catalog and route management. However, now that I think about it we could either:

- change `quick-start` to include catalog and route management OR
- change the process to be 1) run `quick-start` 2) run extra steps if you want to add catalog and route management (a bit more like https://github.com/apache/incubator-openwhisk-devtools/pull/82)

Either way there should be some docs which I can add.

Reply:
👍 on making catalog addition a separate step. I think most people just wanting to try it out won't need the catalogue in the beginning. I'm also working on a PR to automate adding the package providers (alarms, kafka, couchdb) and this will be a separate step as well.

Reply:
OK great, changes incoming...